rosette/lib/util/server.rkt

```diff
@@ -0,0 +1,61 @@
+#lang racket
```

Comment:
I’d call this file streaming-server.rkt or something.

Reply:
Done.